src/main/java/com/rackspace/jenkins_nodepool/NodePoolSlave.java

```diff
@@ -297,6 +318,29 @@ boolean isBuildComplete() {
         }
     }
 
+    /**
+     * A convenience routine to help determine if the builds are finished.
+     *
+     * @param builds a list of builds for this slave
+     * @return true if all the builds are finished, false otherwise
+     */
+    private boolean isAllBuildsFinished(RunList builds) {
```

Comment:
Is the plan to use this in `isBuildComplete?`

eg:
```
if (Iterators.size(builds.iterator()) > 0 && isAllExecutorsIdle(computer.getAllExecutors()) && isAllBuildsFinished(builds) {
```

I can't see any usages of this functions, so it should be used or removed.

Reply:
👍